Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -4591,7 +4592,7 @@ void MainWindow::slotMeChangeStatus(bool /*checked =false */)
     QString statusmsg = ttSettings->value(SETTINGS_GENERAL_STATUSMESSAGE).toString();
     if ((TT_GetFlags(ttInst) & CLIENT_AUTHORIZED) && m_host.statusmsg.size())
         statusmsg = m_host.statusmsg;
-    ChangeStatusDlg dlg(statusmsg, this);
+    ChangeStatusDlg dlg((statusmsg != ttSettings->value(SETTINGS_GENERAL_STATUSMESSAGE)?statusmsg:""), this);

```

Comment:
Isn't there a `.toString()` missing:
`ttSettings->value(SETTINGS_GENERAL_STATUSMESSAGE).toString()`

Reply:
Also why is the `else` empty string? Shouldn't it be `ttSettings->value(SETTINGS_GENERAL_STATUSMESSAGE).toString()` ?

Reply:
My idea is to leave blank value if the current value is the same as global value, so user knows that no specific value is set for this host.